src/__tests__/pretty-dom.js

```diff
@@ -1,25 +1,33 @@
-import {prettyDOM, logDOM} from '../pretty-dom'
-import {getUserCodeFrame} from '../get-user-code-frame'
+/* global globalThis */
+import {prettyDOM as prettyDOMImpl} from '../pretty-dom'
 import {render, renderIntoDocument} from './helpers/test-utils'
 
-jest.mock('../get-user-code-frame')
-
-beforeEach(() => {
-  jest.spyOn(console, 'log').mockImplementation(() => {})
-})
-
-afterEach(() => {
-  console.log.mockRestore()
-})
-
-test('prettyDOM prints out the given DOM element tree', () => {
+function prettyDOM(...args) {
+  let originalProcess
+  // this shouldn't be defined in this environment in the first place
+  if (typeof process === 'undefined') {
+    throw new Error('process is no longer defined. Remove this setup code.')
+  } else {
+    originalProcess = process
+    // TODO: Delete to test browser environments
+    // delete globalThis.process
```

Comment:
Would've caught https://github.com/testing-library/dom-testing-library/issues/1322. Fixing in https://github.com/testing-library/dom-testing-library/pull/1323